lib/views/home/home_view.dart

```diff
@@ -201,8 +201,13 @@ class _HomeViewState extends State<HomeView> {
     return FutureBuilder<List<dynamic>?>(
         future: listPayments(),
         builder: (context, AsyncSnapshot<List<dynamic>?> snapshot) {
-          _checkIfThereAreError(context: context, snapshot: snapshot);
-          if (snapshot.hasData) {
+          String error = '';
```

Comment:
```suggestion
          String? error = null;
```

and why declare this outside of the if? is it not used to store the error message?

Reply:
btw, no empty string, but use the nullable operator, let the language fearture help you.

Reply:
> and why declare this outside of the if? is it not used to store the error message?

Yes it should be inside.Fixing it now.